src/connector.js

```diff
@@ -0,0 +1,769 @@
+	}
+}
+
+if ( !nextPageTemplateHTML ) {
+	if ( lang === "fr" ) {
+		nextPageTemplateHTML = 
+			`<button class="page-button next-page-button">Suivante<span class="wb-inv">: Page suivante des résultats de recherche</span></ button>`;
+	}
+	else {
+		nextPageTemplateHTML = 
+			`<button class="page-button next-page-button">Next<span class="wb-inv">: Next page of search results</span></ button>`;
+	}
+}
+
+if ( !pagerContainerTemplateHTML ) {
+	if ( lang === "fr" ) {
+		pagerContainerTemplateHTML = 
+			`<div class="text-center" >
+				<p class="wb-inv">Pagination des résultats de recherche</p>
+				<ul id="pager" class="pagination mrgn-bttm-0">
+				</ul>
+			</div>`;
+	}
+	else {
+		pagerContainerTemplateHTML = 
+			`<div class="text-center" >
+				<p class="wb-inv">Search results pages</p>
+				<ul id="pager" class="pagination mrgn-bttm-0">
+				</ul>
+			</div>`;
+	}
+}
+
+// auto-create results section if not present
+if ( !resultsSection ) {
+	resultsSection = document.createElement( "section" );
+	resultsSection.setAttribute( "class", "results" );
+
+	baseElement.prepend( resultsSection );
+}
+
+// auto-create query summary element
+if ( !querySummaryElement ) {
+	querySummaryElement = document.createElement( "div" );
+	querySummaryElement.setAttribute( "id", "query-summary" );
+
+	resultsSection.append( querySummaryElement );
+}
+
+// auto-create did you mean element
+if ( !didYouMeanElement ) {
+	didYouMeanElement = document.createElement( "div" );
+	didYouMeanElement.setAttribute( "id", "did-you-mean" );
+
+	resultsSection.append( didYouMeanElement );
+}
+
+// auto-create results section if not present
+if ( !resultListElement ) {
+	resultListElement = document.createElement( "div" );
+	resultListElement.setAttribute( "id", "result-list" );
+
+	resultsSection.append( resultListElement );
+}
+
+// auto-create pager
+if ( !pagerElement ) {
+	let newPagerElement = document.createElement( "div" );
+	newPagerElement.innerHTML = pagerContainerTemplateHTML;
+
+	resultsSection.append( newPagerElement );
+	pagerElement = newPagerElement.querySelector( "#pager" );
+}
+
+// auto-create suggestions element
+if ( !suggestionsElement && params.unsupportedSuggestions && params.numberOfSuggestions > 0 && searchBoxElement ) {
+	suggestionsElement = document.createElement( "section" );
+	suggestionsElement.setAttribute( "id", "suggestions" );
+	suggestionsElement.setAttribute( "class","query-suggestions" );
+
+	searchBoxElement.after( suggestionsElement );
+}
+
+const contextController = buildContext( headlessEngine );
+contextController.set( { "searchPageUrl" : params.originLevel3 } );
+
+// build controllers
+const searchBoxController = buildSearchBox( headlessEngine, {
+	options: {
+		numberOfSuggestions: params.numberOfSuggestions,
+		highlightOptions: {
+			notMatchDelimiters: {
+				open: '<strong>',
+				close: '</strong>',
+			},
+			correctionDelimiters: {
+				open: '<i>',
+				close: '</i>',
```

Comment:
Why to use an element that don't have any semantic? This could be an accessibility issue depending of how it is used.

Did you means here the emphasis element? if so then:

```suggestion
				open: '<em>',
				close: '</em>',
```